python/ray/serve/http_proxy.py

```diff
@@ -22,11 +23,15 @@
     RawASGIResponse,
     receive_http_body,
     Response,
+    set_socket_reuse_port,
 )
 from ray.serve.long_poll import LongPollClient
 
 MAX_REPLICA_FAILURE_RETRIES = 10
 DISCONNECT_ERROR_CODE = "disconnection"
+SOCKET_REUSE_PORT_ENABLED = (
```

Comment:
Does this mean we disable port reuse by default from now on? Users have to explicitly set this env var to reuse ports.

Reply:
Never mind, I just noticed the default.